spec/models/comment_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'A beautiful beach',
+              post_counter: 0)
+  post = Post.create(user_id: user.id, title: 'Hello', text: 'This is my first post', likes_counter: 0,
+                     comments_counter: 0)
```

Comment:
- [ ]  The variable `user` used at line 6 is not defined, making the tests fail. Please assign the user created at line 4 to the `user` variable.
```rb
user = User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'A beautiful beach',
              post_counter: 0)
```

Reply:
resolved